test/validators/array.spec.js

```diff
@@ -0,0 +1,25 @@
+import { expect } from 'chai';
+import validator from '../../lib';
+
+describe('String validator', () => {
+  const rules = {
+    someField: ['array']
+  };
+
+  it('should success with valid input', () => {
```

Comment:
i think you miss the isNil coverage here should add the undefined and null as well

Reply:
thanks for reminding :)